### PR TITLE
Merge up 4.10.x into 5.3.x

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: acceptance/Gemfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ def location_for(place, fake_version = nil)
 end
 
 gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.33')
-gem "beaker-puppet", *location_for(ENV['BEAKER_PUPPET_VERSION'] || "~> 0.10")
+gem "beaker-puppet", *location_for(ENV['BEAKER_PUPPET_VERSION'] || "~> 0.11")
```
Comment (melissa) [inline acceptance/Gemfile]: We need to get a new beaker-puppet release to fix a bug on Japanese Windows before this should be merged up

**Comment by branan** [comment]
I'd like to see us get 1.10.x green just to be sure we've got all the kinks worked out before we merge this up.

**Comment by melissa** [comment]
Should I rekick the 1.10.x pipeline, or wait until it runs through tonight?

**Comment by melissa** [comment]
This has been updated to include the latest contents of 4.10.x that should run through ci tonight

**Comment by geoffnichols** [comment]
👍

**Pull request merged**